proto/cosmos/vesting/v1beta1/tx.proto

```diff
@@ -43,8 +48,39 @@ message MsgCreatePeriodicVestingAccount {
   string   to_address                      = 2 [(gogoproto.moretags) = "yaml:\"to_address\""];
   int64    start_time                      = 3 [(gogoproto.moretags) = "yaml:\"start_time\""];
   repeated Period vesting_periods = 4 [(gogoproto.nullable) = false];
+  bool merge = 5;
 }
 
-// MsgCreatePeriodicVestingAccountResponse defines the Msg/CreatePeriodicVestingAccount
+// MsgCreatePeriodicVestingAccountResponse defines the MsgCreatePeriodicVestingAccount
 // response type.
 message MsgCreatePeriodicVestingAccountResponse {}
+
+// MsgCreateTrueVestingAccount defines a message that enables creating a true vesting account.
+message MsgCreateTrueVestingAccount {
+  option (gogoproto.equal) = false;
+
+  string from_address = 1 [(gogoproto.moretags) = "yaml:\"from_address\""];
+  string   to_address                      = 2 [(gogoproto.moretags) = "yaml:\"to_address\""];
```

Comment:
Since we decided that creating a `ClawbackVestingAccount` (revised name, renaming commit will follow) is a non-privileged operation, we don't want to use the term "admin", so we're describing the one making the grant (and signing it) as the "funder". Added descriptions in all the request message fields to make things more clear.